### ISSUE TITLE
[Page Add][Basic Function] Add one project board on Flux.ai platform and recording the steps

### ISSUE THREAD

**Issue by MatthewJeffson** [Seeed-Studio/wiki-documents#1902]
## Overview
- We need your help to add required Boards in the [Flux.ai](https://www.flux.ai/seeedstudio):
          - [WLED on XIAO ESP32](https://www.instructables.com/WLED-ON-XIAO-ESP32/)

![Image](https://github.com/user-attachments/assets/9d7bf411-e481-4129-a809-ea87b9d82963)

Reference:
- [XIAO SAMD21 on Flux.ai(by amazing cnaville89)](https://www.flux.ai/cnaville89/seeed-xiao-samd21?editor=pcb_3d)
- [Move your KiCAD parts into Flux](https://www.youtube.com/watch?v=eplpLndGfvk)

## How do we work together?
- **Leave a comment with your GitHub name below** as you accept this assignment and will complete the contents about what we are trying to present.
- You can refer to the [Contributor Program](https://wiki.seeedstudio.com/Contributor/) page for more details receiving.
- We have presented two ways to [PR to this repo](https://wiki.seeedstudio.com/Contribution-Guide/).
- You will be invited as our **"Contributor"** once you ["Pull Request"](https://wiki.seeedstudio.com/Contribution-Guide/).
- The maintainer will review the PR and then merge.
## What do we offer?
We want to express our gratitude to our contributors by offering a range of rewards.  Depending on the difficulties(Tier 0/1/2/3), the time we expect(Days 1/3/7/15),  the actual  submission time and the actual submission contents, we will offer different levels of rewards.
1. We will provide our products or cash bonus to our contributors as a token of appreciation.
2. For those contributors who join our ["Ranger Program"](https://wiki.seeedstudio.com/ranger/), we offer even more exciting rewards, including access to our prototype products.
3. In addition, all contributors and their contributions will be acknowledged and showcased on our wiki platform.
## FAQ
- For any question, the contributor can directly commend here.
### I think I can complete the assignment, but I need more time?
- The timeframe here is based on our old experience, from people who has done it or something like that.
- However, **many projects will take much more time** under actual circumstances (including the main language of the contributors receiving the task and the actual creative difficulty)
- You can leave the time you need and a brief reason, we will evaluate the situation accordingly.

**Comment by gokuxmaker** [comment]
Completed  
Flux.ai project link: https://www.flux.ai/seeedstudio/wled-rgb-led-driverp  
Part 1 of the design tutorial video: https://www.youtube.com/watch?v=BQ12ifMhkRw  
Part 2 of the build and test video: https://www.youtube.com/watch?v=ZayS9bxdt98&t  
![Image](https://github.com/user-attachments/assets/09b13cad-0fe2-4a1a-81b4-771694644618)
Additionally, a written blog and 3D print files will be available through a link in the Part 2 video .

**Comment by Dangerousfish** [comment]
Notifying @MatthewJeffson

**Comment by MatthewJeffson** [comment]
Yes! Thank you @Dangerousfish for the notification! And thanks @gokuxmaker Gokul for the project added!
Best Regards,
Matthew
